src/main/java/exception/ExceptionCode.java

```diff
@@ -0,0 +1,7 @@
+package exception;
+
+public interface ExceptionCode {
+
+  String getDefaultMessage();
```

Comment:
```suggestion
  String getMessage();
```
는 어떨까요? 인터페이스 입장에서 Default 를 알필요 보다는 정말로 전달해야할 메시지가 중요할 것 같아서요 🤔